setup.py

```diff
@@ -32,9 +32,9 @@ def get_readme():
     packages=find_packages(),
     zip_safe=False,
     install_requires=[
-        # avoid django 1 <1.11.28 and django 2 <2.2.10
+        # avoid django 1 <1.11.28 and django 2 <2.2.10 and django 3 < 3.0.3
         # https://github.com/advisories/GHSA-hmr4-m2h5-33qx
-        'django>=1.11.28,!=2.0.*,!=2.1.*,!=2.2.0,!=2.2.1,!=2.2.2,!=2.2.3,!=2.2.4,!=2.2.5,!=2.2.6,!=2.2.7,!=2.2.8,!=2.2.9,<3',
+        'django>=1.11.28,!=2.0.*,!=2.1.*,!=2.2.0,!=2.2.1,!=2.2.2,!=2.2.3,!=2.2.4,!=2.2.5,!=2.2.6,!=2.2.7,!=2.2.8,!=2.2.9,!=3.0.0,!=3.0.1,!=3.0.2',
```

Comment:
I think we'll still need a <4 here so that when it comes out in the future, it's not retroactively considered a valid version (until we support it).

Reply:
Agreed.